jwt-authorizer/templates/redis-deployment.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: redis
+  name: redis
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: redis
+  template:
+    metadata:
+      labels:
+        app: redis
+    spec:
+      containers:
+      - name: redis
+        image: redis
+        imagePullPolicy: Always
+        args:
+        - redis-server
+        - --appendonly yes
+        ports:
+        - containerPort: 6379
+        resources:
+          limits:
+            cpu: "1"
+        volumeMounts:
+        - name: redis-data
+          mountPath: "/data"
+      volumes:
+      - name: redis-data
+        emptyDir: {}
```

Comment:
If we're just doing emptydir, why have it as a volume at all?  Or is the plan to make this configurable and store this on a volume?

Reply:
yeah I need to fix this... I plan on doing that this week or next because I think we got a volume thing figured out at NCSA

Reply:
I fixed this - I really don't understand why github is showing the old data 🤯